shared/src/main/java/com/faforever/neroxis/mask/BooleanMask.java

```diff
@@ -1008,11 +1028,21 @@ public BooleanMask dilute(float strength, int count) {
             int size = getSize();
             for (int i = 0; i < count; i++) {
                 long[] maskCopy = getMaskCopy();
-                applyWithSymmetry(symmetryType, (x, y) -> {
-                    if (!getPrimitive(x, y) && random.nextFloat() < strength && isEdge(x, y)) {
-                        applyAtSymmetryPoints(x, y, symmetryType, (sx, sy) -> setBit(sx, sy, true, size, maskCopy));
-                    }
-                });
+
+                if (symmetrySettings.getSymmetry(SymmetryType.SPAWN).isOddSymmetry()) {
+                    apply((x, y) -> {
+                        if (!getPrimitive(x, y) && random.nextFloat() < strength && isEdge(x, y)) {
+                            setBit(x, y, true, size, maskCopy);
+                        }
+                    });
```

Comment:
This ends up breaking the symmetry guarantees because a new random is used for every x,y